src/components/Features.tsx

```diff
@@ -145,7 +145,7 @@ export function Features() {
     <section
       id='how-it-works'
       aria-label='Features for simplifying everyday business tasks'
-      className='bg-white pt-20 pb-14 sm:pb-20 sm:pt-32 lg:pb-32'
```

Comment:
Removing `pb-14` makes sense, but why add `pb-4`?

Reply:
It creates an equal distance in the title or heading, below and above. One finger apart, equally

Reply:
![Screenshot 2022-10-30 174331](https://user-images.githubusercontent.com/82657928/198887838-4b659e8a-4abe-4e9e-bffd-9a12d069f932.png)
It removes the extra padding after the `<hr>` tag or line divider, makes it much neater to something like this:

![Screenshot 2022-10-30 174534](https://user-images.githubusercontent.com/82657928/198887936-9e04e90c-f892-4957-ae0c-927f20914e43.png)

As I've mentioned before, the screenshots of the site and the code itself are provided for you to see the changes in correlation to why/ and where the changes have been made :)

Reply:
Sorry, the preview of this project is not quite working at the moment, that's why I can't see the deployed version.

> It removes the extra padding after the `<hr>` tag or line divider, makes it much neater to something like this:

Yeah, I got that part. My question was why add `pb-4`, wouldn't it add some extra padding again?

Reply:
No, no extra padding at all.  Why? I reduced the padding from `pb-14` to `pb-4` for the reason you've quoted from me :)

Reply:
Ended up cloning your repo. It shows this padding that I was referring to :point_down:

![screenshot-showing-extra-space](https://user-images.githubusercontent.com/108616679/199223728-42f64f14-80e1-4cff-90bf-98c205bf891e.png)

Reply:
Hello there, forgive my late response. I guess i will revert the changes then and leave that bottom as is.

Reply:
We're all working on it in our spare time, so don't worry. I appreciate whatever contribution you make.

Going back to the padding issue. My point was, you should probably remove the padding completely instead of just reducing it. I think that's what you intended to do in the first place.

Reply:
Thank you very much

Okay, to remove the padding completely, got it! Tbh, I'm not sure what I intended to do 😂

Reply:
Haha no worries. Thanks for working on it.